sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/MultiPathImporter.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.php.phpunit;
+
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.sonar.api.batch.sensor.SensorContext;
+import org.sonar.api.batch.sensor.coverage.CoverageType;
+import org.sonar.api.measures.CoreMetrics;
+import org.sonar.plugins.php.PhpPlugin;
+
+/**
+ * Class is used to save coverage results from list of PHPUnit reports provided in {@link PhpPlugin#PHPUNIT_COVERAGE_REPORT_PATHS_KEY} property.
+ */
+public class MultiPathImporter extends CoverageResultImporter {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(MultiPathImporter.class);
+
+  private static final String MSG = "coverage";
+  private static final String PATHS_KEY = PhpPlugin.PHPUNIT_COVERAGE_REPORT_PATHS_KEY;
+
+  protected MultiPathImporter() {
+    // CoverageType.UNIT is fictitious. It will be ignored.
+    super(PhpPlugin.PHPUNIT_COVERAGE_REPORT_PATHS_KEY, MSG, CoreMetrics.LINES_TO_COVER, CoreMetrics.UNCOVERED_LINES, CoverageType.UNIT);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "CoverageType"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "UNIT"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)